src/streamlink/plugins/cinergroup.py

```diff
@@ -40,8 +69,13 @@ def _get_streams(self):
                 validate.get("ht_stream_m3u8"),
             ),
         ))
-        if stream_url:
-            return HLSStream.parse_variant_playlist(self.session, stream_url)
+        return live_url
+
+    def _get_streams(self):
+        live_url = self._get_live_url()
+        if not live_url:
+            live_url = self._get_live_url2()
```

Comment:
Parse the HTML and pass the root node as an argument to the different methods. And don't forget to return `None` if the `live_url` is `None`. Otherwise this will lead to an error in the `HLSStream.parse_variant_playlist()` call.

```suggestion
        root = self.session.http.get(self.url, schema=validate.Schema(
            validate.parse_html(),
        ))
        live_url = self._get_live_url(root) or self._get_live_url2(root)
        if not live_url:
            return
```